docs/UserGuide.adoc

```diff
@@ -16,7 +16,9 @@ By: `Team PIMP (W10-B1)`      Since: `Sep 2016`      Licence: `MIT`
 
 == Introduction
 
-To be filled.
+PIMP (Personal Information Manager Program) is a command line interface (CLI) address book that allows you to store your contacts. You can use PIMP to add friends, add specific tags to them, add their social media accounts (such as Facebook, Twitter, Instagram), or add remarks. You can search through the list of your contacts by typing specific names. For more advanced user, you can use regex to find your friends. Most of the functionality works by typing in single line commands.
+
+PIMP v1.5 is available on Windows platform running Java 8.
```

Comment:
Shouldn't this be "any platform running Java 8"?

Reply:
Yeah you're right, let me change